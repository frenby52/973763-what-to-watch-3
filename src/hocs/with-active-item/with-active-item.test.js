import React from "react";
import renderer from "react-test-renderer";
import withActiveItem from "./with-active-item.js";

const MockComponent = () => <div />;

const MockComponentWrapped = withActiveItem(MockComponent);

it(`withActiveItem HOC is rendered correctly`, () => {
  const tree = renderer.create(<MockComponentWrapped />).toJSON();

  expect(tree).toMatchSnapshot();
});
