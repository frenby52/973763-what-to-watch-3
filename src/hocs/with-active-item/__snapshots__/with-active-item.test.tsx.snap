// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveItem HOC is rendered correctly 1`] = `<div />`;
